app/posts/ecf-v2/non-state-funded-schools-in-RECT.md

```diff
@@ -0,0 +1,35 @@
+---
+title: No longer asking non-state-funded schools to use the RECT service
+description: "Why we’ve changed which types of schools can access the service."
+date: 2025-03-13
```

Comment:
I wonder if we can just say 'unfunded' schools, or schools that are not funded for ECF.